Cargo.toml

```diff
@@ -40,14 +40,12 @@ png = { version = "0.17" }
 tiny_http = "0.12"
 
 [build-dependencies]
-auto_generate_cdp = { version = "0.3.4", default-features = false }
+auto_generate_cdp = "0.4.0"
 
 [lib]
 name = "headless_chrome"
 path = "src/lib.rs"
 
 [features]
-default = ["auto_generate_cdp/native-tls"]
 fetch = ["ureq", "directories", "zip", "walkdir"]
-rustls = ["auto_generate_cdp/rustls"]
```

Comment:
we are getting rid of this because of ureq right ? correct me if im wrong

Reply:
That's correct, it wasn't enabling anything else than the optional reqwest dependencies on auto_generate_cdp side

Reply:
auto_generate_cdp would need a new crates.io release before this compiles though, I can update the PR if you choose something else than 0.4.0 for that

Reply:
> That's correct, it wasn't enabling anything else than the optional reqwest dependencies on auto_generate_cdp side

cool makes sense

Reply:
> auto_generate_cdp would need a new crates.io release before this compiles though, I can update the PR if you choose something else than 0.4.0 for that

no 0.4.0 sounds good let me publish the release